### PR TITLE
Add Multi Sample Running & Fix result image channel

### PR DESCRIPTION
**Pull request by feelgom**
I added multi samples running code.
You can input the smaple images' directory through the input argument '-ss'.

Also, in function 'plot_result_multi', an error occurred in the cv2 image channel and I corrected it.

### CODE REVIEW

**Comment by feelgom** [comment]
please merge PR

File: qatm_pytorch.py [hunk 1]
```diff
@@ -294,9 +294,9 @@ def plot_result_multi(image_raw, boxes, indices, show=False, save_name=None, col
     for i in range(len(indices)):
         d_img = plot_result(d_img, boxes[i][None, :,:].copy(), color=color_list[indices[i]])
     if show:
-        plt.imshow(d_img)
+        plt.imshow(d_img[:,:,::-1])
```
Comment (Arka161) [inline qatm_pytorch.py:297]: Will keeping only `d_img` break the code now?

File: qatm.py [hunk 2]
```diff
@@ -1,9 +1,13 @@
 from pathlib import Path
 import torch
+import torch.nn as nn
```
Comment (Arka161) [inline qatm.py:3]: This import is not needed.

File: qatm.py [hunk 3]
```diff
@@ -32,21 +36,49 @@
...
+            print("calculate score...")
+            scores, w_array, h_array, thresh_list = run_multi_sample(model, dataset)
+            print("nms...")
+            boxes, indices = nms_multi(scores, w_array, h_array, thresh_list)
+            d_img = plot_result_multi(dataset.image_raw, boxes, indices, show=True, save_name=os.path.join(result_path,image_name)+'.png')
+            print("result image was saved")
+            del(dataset)
+            del(d_img)
+            gc.collect()
+            torch.cuda.empty_cache()
```
Comment (Arka161) [inline qatm.py:83]: Isn't it more efficient if the cache is cleared outside the `for` loop?

**Pull request merged**